Azure/azure-ad/_meta/fields.yml

```
@@ -3,236 +3,299 @@ action.properties:
   name: azuread.properties.deviceDetail.isCompliant
   type: bool
+
 azuread.properties.deviceDetail.isManaged:
-  description: ""
+  description: ''
   name: azuread.properties.deviceDetail.isManaged
   type: bool
+
 azuread.properties.deviceDetail.trustType:
-  description: ""
+  description: ''
   name: azuread.properties.deviceDetail.trustType
   type: keyword
+
 azuread.properties.id:
-  description: ""
+  description: ''
   name: azuread.properties.id
   type: keyword
+
 azuread.properties.requestId:
-  description: ""
+  description: ''
   name: azuread.properties.requestId
   type: keyword
+
 azuread.properties.riskDetail:
-  description: ""
+  description: ''
   name: azuread.properties.riskDetail
   type: keyword
+
 azuread.properties.riskEventType:
-  description: ""
+  description: ''
   name: azuread.properties.riskEventType
   type: keyword
+
 azuread.properties.riskEventTypes:
   description: riskEventTypes
   name: azuread.properties.riskEventTypes
   type: keyword
+
 azuread.properties.riskEventTypes_v2:
   description: riskEventTypes_v2
   name: azuread.properties.riskEventTypes_v2
   type: keyword
+
 azuread.properties.riskLevel:
-  description: ""
+  description: ''
   name: azuread.properties.riskLevel
   type: keyword
+
 azuread.properties.riskLevelAggregated:
   description: riskLevelAggregated
   name: azuread.properties.riskLevelAggregated
   type: keyword
+
 azuread.properties.riskLevelDuringSignIn:
   description: riskLevelDuringSignIn
   name: azuread.properties.riskLevelDuringSignIn
   type: keyword
+
 azuread.properties.riskState:
-  description: ""
+  description: ''
   name: azuread.properties.riskState
   type: keyword
+
 azuread.properties.source:
-  description: ""
+  description: ''
   name: azuread.properties.source
   type: keyword
+
 azuread.properties.status.additionalDetails:
-  description: ""
+  description: ''
   name: azuread.properties.status.additionalDetails
   type: keyword
+
 azuread.properties.status.errorCode:
-  description: ""
+  description: ''
   name: azuread.properties.status.errorCode
   type: keyword
+
 azuread.properties.status.failureReason:
-  description: ""
+  description: ''
   name: azuread.properties.status.failureReason
   type: keyword
+
 azuread.properties.targetDevicePrincipalName:
-  description: ""
+  description: ''
   name: azuread.properties.targetDevicePrincipalName
   type: keyword
+
 azuread.properties.targetUserPrincipalName:
-  description: ""
+  description: ''
   name: azuread.properties.targetUserPrincipalName
   type: keyword
+
 azuread.resourceId:
-  description: ""
+  description: ''
   name: azuread.resourceId
   type: keyword
+
 azuread.resourceTenantId:
-  description: ""
+  description: ''
   name: azuread.resourceTenantId
   type: keyword
+
 azuread.resultSignature:
-  description: ""
+  description: ''
   name: azuread.resultSignature
   type: keyword
+
+azuread.riskReasons:
+  description: ''
+  name: azuread.riskReasons
+  type: array
```

in order to be consistent with the previous comment
```suggestion
azuread.properties.riskReasons:
  description: ''
  name: azuread.properties.riskReasons
  type: array
```